codecrafters/seed.py

```diff
@@ -0,0 +1,151 @@
+            "email": "student1@email.com",
+            "role": "student",
+        },
+        {
+            "userId": str(uuid.uuid4()),
+            "userName": "teacher1",
+            "fullName": "Teacher 1",
+            "email": "teacher1@email.com",
+            "role": "teacher",
+        },
+    ]
+    # Insert Users
+    for user in users:
+        table.put_item(
+            Item={
+                "PK": f"USER#{user['userId']}",
+                "SK": f"METADATA",
+                "userName": user["userName"],
+                "fullName": user["fullName"],
+                "email": user["email"],
+                "role": user["role"]
+            }
+        )
+
+    # Sample Quizzes
+    quizzes = [
+        {
+            "quizId": quizId,
+            "title": "Python Basics",
+            "description": "A quiz on basic Python concepts.",
+        },
+        {
+            "quizId": str(uuid.uuid4()),
+            "title": "AWS Fundamentals",
+            "description": "A quiz on AWS core services.",
+        },
+    ]
+    # Insert Quizzes
+    for quiz in quizzes:
+        table.put_item(
+            Item={
+                "PK": f"QUIZ#{quiz['quizId']}",
+                "SK": f"METADATA",
+                "title": quiz["title"],
+                "description": quiz["description"],
+            }
+        )
+
+    # Sample Quizzes
+    questions = [
+        {
+            "quizId": quizId,
+            "questionId": str(uuid.uuid4()),
+            "questionText": "What is the keyword used to define a function in Python?",
+            "type": "single choice",
+            "options": ["function", "def", "func", "define"],
+            "correctAnswer": "def",
+        }
+    ]
+    # Insert Questions for each Quiz
+    for question in questions:
+        table.put_item(
+            Item={
+                "PK": f"QUIZ#{question['quizId']}",
+                "SK": f"QUESTION#{question['questionId']}",
+                "questionText": question["questionText"],
+                "type": question["type"],
+                "options": question["options"],
+                "correctAnswer": question["correctAnswer"],
+            }
+        )
+
+    # Sample UserAnswers
+    userAnswers = list(
+        map(
+            lambda question: {
+                "PK": f"USER#{userId}#ATTEMPT#{attemptId}#QUIZ#{quizId}",
+                "SK": f"QUESTION#{question['questionId']}",
+                "userAnswer": "def",
+                "status": "pass",
+                "timestamp": str(datetime.now()),
+            },
+            questions,
+        )
+    )
+    # Insert UserAnswers
+    for userAnswer in userAnswers:
+        table.put_item(
+            Item={
+                "PK": userAnswer["PK"],
+                "SK": userAnswer["SK"],
+                "userAnswer": userAnswer["userAnswer"],
+                "status": userAnswer["status"],
+                "timestamp": userAnswer["timestamp"],
+            }
+        )
+
```

Comment:
Now it makes sense what you mentioned in the meeting that we don't need a history, good point! 👍